sdk/iot/core/tests/cmocka/main.c

```diff
@@ -104,6 +114,16 @@ int main()
 
     // IoT Hub Client
     cmocka_unit_test(test_az_iot_hub_client_get_default_options_succeed),
+    cmocka_unit_test(test_az_iot_hub_client_init_succeed),
+    cmocka_unit_test(test_az_iot_hub_client_init_custom_options_succeed),
+    cmocka_unit_test(test_az_iot_hub_client_user_name_get_succeed),
+    cmocka_unit_test(test_az_iot_hub_client_user_name_get_small_buffer_fail),
+    cmocka_unit_test(test_az_iot_hub_client_user_name_get_user_options_succeed),
+    cmocka_unit_test(test_az_iot_hub_client_user_name_get_user_options_small_buffer_fail),
+    cmocka_unit_test(test_az_iot_hub_client_id_get_succeed),
+    cmocka_unit_test(test_az_iot_hub_client_id_get_small_buffer_fail),
+    cmocka_unit_test(test_az_iot_hub_client_id_module_get_succeed),
+    cmocka_unit_test(test_az_iot_hub_client_id_module_get_small_buffer_fail),
```

Comment:
this is becoming unmanageable

Can we create a variable for each group? and then pass to group runner?

Reply:
Will be discussed in the SDK meeting today. I am continuing with the way the IoT SDK has written unit tests to not block and will reformat the tests as decided on by the SDK at large. Agreed though.